### ISSUE TITLE
Problem with debugger

### ISSUE THREAD

**Issue by drmeister** [drmeister/clasp#154]
stassats reports:
```
Debugger received error of type: SIMPLE-FILE-ERROR In file_truename ../../src/core/unixfsys.cc line 443 COMMON-LISP:FILE-ERROR :initializers (:PATHNAME "stas@debian.12018:/home/stas/1400411332"  )
```

**Comment by Shinmera** [comment]
Is this related to #151?

**Comment by stassats** [comment]
I don't think so, my build directory is still intact.

**Comment by drmeister** [comment]
I think this is fixed now.

**Comment by Shinmera** [comment]
How do I reproduce this? Is this the symlink thing?

**Comment by Shinmera** [comment]
Fixed on 712260b4038b40e04d8928b7183e2af5da416a0d
```
╭─linus@linut ~/autobuild/clasp-testing/.autobuild/712260b4038b40e04d8928b7183e2af5da416a0d/build/clasp/bin  ‹712260b*› 
╰─➤  ln -s /blablabla ~/foo
╭─linus@linut ~/autobuild/clasp-testing/.autobuild/712260b4038b40e04d8928b7183e2af5da416a0d/build/clasp/bin  ‹712260b*› 
╰─➤  ./clasp_boehm_o 
Starting Boehm cclasp 0.3-test-10-480-g712260b ... loading image... it takes a few seconds
Top level.
> (foo)
;;; Warning: Condition: Undefined function named COMMON-LISP-USER::FOO

Condition of type: SIMPLE-PROGRAM-ERROR
The symbol FOO has no function bound to it

Available restarts:
(use :r1 to invoke restart 1)

1. (RESTART-TOPLEVEL) Go back to Top-Level REPL.

Broken at frame[117] TOP-LEVEL.
 File: #P"/home/linus/.cache/shirakumo/autobuild/clasp-testing/.autobuild/712260b4038b40e04d8928b7183e2af5da416a0d/build/clasp/Contents/Resources/lisp/kernel/lsp/top.lsp" (Position #21029)
>> 
```
